gmplot/gmplot.py

```diff
@@ -45,24 +51,37 @@ def geocode(self, location_string):
     def grid(self, slat, elat, latin, slng, elng, lngin):
         self.gridsetting = [slat, elat, latin, slng, elng, lngin]
 
-    def marker(self, lat, lng, color='#FF0000', c=None):
+    def marker(self, lat, lng, color='#FF0000', c=None, title=None, defaultMarker=None, **kwargs):
         if c:
             color = c
         color = self.color_dict.get(color, color)
         color = self.html_color_codes.get(color, color)
-        self.points.append((lat, lng, color[1:]))
+        if title!=None:
+            self.points.append((lat, lng, color[1:], title))
+        else:
+            self.points.append((lat, lng, color[1:]))
 
-    def scatter(self, lats, lngs, color=None, size=None, marker=True, c=None, s=None, **kwargs):
+    def scatter(self, lats, lngs, color=None, size=None, marker=True, c=None, s=None, titles=None, defaultMarker=None, **kwargs):
         color = color or c
         size = size or s or 40
         kwargs["color"] = color
         kwargs["size"] = size
-        settings = self._process_kwargs(kwargs)
-        for lat, lng in zip(lats, lngs):
-            if marker:
-                self.marker(lat, lng, settings['color'])
-            else:
-                self.circle(lat, lng, size, **settings)
+        kwargs["defaultMarker"] = defaultMarker
```

Comment:
It would be great to move any mention of this defaultMarker into _process_kwargs since that is where the default is really stored. From `kwargs.get("defaultMarker", None)`, the default is really `None`.